tests/Unit/Evolution/Systems/Cce/BoundaryTestHelpers.hpp

```diff
@@ -41,7 +41,7 @@ Tensor<ComplexModalVector, Structure...> tensor_to_goldberg_coefficients(
 template <typename... Structure>
 Tensor<ComplexModalVector, Structure...> tensor_to_libsharp_coefficients(
     const Tensor<DataVector, Structure...>& nodal_data,
-    const size_t l_max) noexcept {  // NOLINT
+    const size_t l_max) noexcept {  // NOLINT(clang tidy error)
```

Comment:
sorry for not being clear, can you put in the actual "error name" that clang-tidy gave you, such as `google-explicit-constructor`